app/lib/transformer/TransformerFactory.php

```diff
@@ -50,19 +50,7 @@ public static function makeStatic($name)
      */
     public function make($name)
     {
-        // transformers ares stateless we can reuse them as we like
-        if (isset(self::$runtimeCache[$name])) {
-            return self::$runtimeCache[$name];
-        }
-
-        if (!$this->classExists(__NAMESPACE__, $name)) {
-            return null;
-        }
-        $className = __NAMESPACE__ . '\\' . $name;
-
-        self::$runtimeCache[$name] = new $className();
```

Comment:
Now makeStatic does not create a *new* instance which was the 1 percent difference